weblab/experiments/templatetags/experiments.py

```diff
@@ -59,6 +59,19 @@ def url_version_comparison_matrix(entity):
     return reverse('experiments:list', kwargs=kwargs)
 
 
+@register.simple_tag
+def url_protocol_model_comparison_matrix(experiment):
+    kwargs = {}
+
+    kwargs['model_pks'] = '/%d' % experiment.model.pk
+    kwargs['model_versions'] = '/*'
```

Comment:
This doesn't quite work because you get URLs looking like `/experiments/models/1/versions/*protocols/42/versions/*`, i.e. with no slash between the `*` and `protocols`. I suspect just setting model_versions to `/*/` would resolve this!

Reply:
But we should also add a test that the templatetag produces expected output given particular arguments, probably in `experiments/tests/test_templatetags.py` `test_url_version_comparison_matrix`

Reply:
good points thanks